sci-visualization/fityk/fityk-1.3.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit fdo-mime python-r1 wxwidgets
+
+DESCRIPTION="General-purpose nonlinear curve fitting and data analysis"
+HOMEPAGE="http://fityk.nieto.pl/"
+SRC_URI="https://github.com/wojdyr/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="gnuplot nlopt readline python static-libs wxwidgets"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="
+	>=sci-libs/xylib-1
+	>=dev-lang/lua-5.1:0
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline:* )
+	wxwidgets? ( x11-libs/wxGTK:3.0 )"
+DEPEND="${CDEPEND}
+	dev-libs/boost:=
+	dev-lang/swig"
+RDEPEND="${CDEPEND}
+	gnuplot? ( sci-visualization/gnuplot )"
+
+src_prepare() {
+	use python && python_copy_sources
+	default
+}
+
+src_configure() {
+	common_confargs=(
+		--with-wx-config=wx-config-3.0
+	)
+
+	econf \
+		${common_confargs[@]} \
+		--disable-python \
+		$(use_enable nlopt) \
+		$(use_enable wxwidgets GUI) \
+		$(use_with readline) \
+		$(use_enable static-libs static)
+	if use python; then
+		python_configure() {
+			cd "${BUILD_DIR}" || die
+			econf \
+				${common_confargs[@]} \
+				--enable-python \
+				--disable-nlopt \
+				--disable-GUI \
+				--without-readline
+		}
+		python_foreach_impl python_configure
+	fi
+}
+
+src_compile() {
+	default
+	if use python; then
+		python_compilation() {
+			emake -C "${BUILD_DIR}"/fityk swig/_fityk.la
+		}
+		python_foreach_impl python_compilation
+	fi
+}
+
+src_install() {
+	default
+	if use python; then
+		python_installation() {
+			emake DESTDIR="${ED%/}" -C "${BUILD_DIR}"/fityk install-pyexecLTLIBRARIES
+			rm "${ED%/}"/$(python_get_sitedir)/*.la || die
```

Comment:
I think in this case they are needed. There is no pc file and there is a libfityk.a and libfityk.la says that there are dependencies